test/fork/withdraw-enzyme-shares.js

```diff
@@ -0,0 +1,124 @@
+const { ethers, network } = require('hardhat');
+const { expect } = require('chai');
+const { parseEther, toUtf8Bytes } = ethers.utils;
+const evm = require('./evm')();
+const { enableAsEnzymeReceiver } = require('./utils');
+const { toBytes8 } = require('../../lib/helpers');
+
+const WETH_ADDRESS = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2';
+const ENZYMEV4_VAULT_PROXY_ADDRESS = '0x27F23c710dD3d878FE9393d93465FeD1302f2EbD';
+const ENZYME_FUND_VALUE_CALCULATOR_ROUTER = '0x7c728cd0CfA92401E01A4849a01b57EE53F5b2b9';
+const COWSWAP_SETTLEMENT = '0x9008D19f58AAbD9eD0D60971565AA8510560ab41';
+const ENZYMEV4_VAULT_PRICE_FEED_ORACLE_AGGREGATOR = '0xCc72039A141c6e34a779eF93AEF5eB4C82A893c7';
+
+const V2Addresses = {
+  SwapOperator: '0xcafea536d7f79F31Fa49bC40349f6a5F7E19D842',
+};
+
+const getSigner = async address => {
+  const provider =
+    network.name !== 'hardhat' // ethers errors out when using non-local accounts
+      ? new ethers.providers.JsonRpcProvider(network.config.url)
+      : ethers.provider;
+  return provider.getSigner(address);
+};
+
+describe('prevent switch or withdraw membership when tokens are locked', function () {
+  before(async function () {
+    // Initialize evm helper
+    await evm.connect(ethers.provider);
+    await getSigner('0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9');
```

Comment:
is this a random address? or the swap controller?

Reply:
It is pretty much random, checked with @shark0der  and it was probably testing the getSigner function and i just copy pasted it along with the whole before block

removed from everywhere now